lib/sidekiq_prometheus/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module SidekiqPrometheus
-  VERSION = "2.0.1"
+  VERSION = "2.1.0"
```

Comment:
```suggestion
  VERSION = "2.0.1.1"
```

Just so that we don't break the sequence with upstream